client/my-sites/domains/components/domain-warnings/index.jsx

```diff
@@ -420,6 +430,74 @@ export class DomainWarnings extends React.PureComponent {
 		);
 	};
 
+	newTransfers = domains =>
+		domains.filter(
+			domain =>
+				domain.registrationMoment &&
+				moment( domain.registrationMoment )
+					.add( 3, 'days' )
+					.isAfter( moment() ) &&
+				domain.transferStatus === transferStatus.COMPLETED
+		);
+
+	loadNameServers = domains => each( domains, domain => fetchNameservers( domain.name ) );
+
+	nameServersLoaded = domains =>
+		every( domains, domain => NameserversStore.getByDomainName( domain.name ).hasLoadedFromServer );
+
+	newTransfersWrongNS = () => {
+		const { newTransfers } = this.state;
+
+		if ( newTransfers.length === 0 || ! this.nameServersLoaded( newTransfers ) ) {
+			return null;
+		}
+
+		const newTransfersOtherNS = newTransfers.filter(
+			domain => ! isWpcomDefaults( NameserversStore.getByDomainName( domain.name ) )
+		);
+
+		const domain = newTransfersOtherNS[ 0 ].name;
+
+		const { translate } = this.props;
+		const compactMessage = translate( 'Update nameservers' );
+		const domainManagementNameserversLink = domainManagementNameServers(
+			this.props.selectedSite.slug,
+			domain
+		);
+
+		const actionText = translate( 'Info', {
```

Comment:
:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?